plexmuxy/font.py

```
@@ -0,0 +1,180 @@
+    destination: Path,
+    media_config: MediaConfig,
+    font_config: FontConfig,
+) -> list[Path]:
+    suffix = archive.suffix.lower()
+    if suffix == ".zip":
+        names = preview_zip_names(archive)
+    elif suffix == ".7z":
+        names = preview_7z_names(archive)
+    elif suffix == ".rar":
+        if not font_config.unrar_path:
+            raise ValueError("Unrar path is not configured")
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)